### DISCUSSION TITLE
Running Input Files in Tests Folder Error: no variable 'dom0Scale'/'dom1Scale' found for substitution expression

### DISCUSSION THREAD

**Discussion by ghost** [shannon-lab/zapdos#88] [Q&A Getting Started] [answered]
I'm trying to run some of the input files within the tests folder to get a feel for Zapdos, but every time I try to run them through peacock I get the same error that "no variable 'dom0Scale'/'dom1Scale' found for substitution expression". Even when I execute the input file with zapdos-opt to get a .e file, Peacock doesn't recognize the transient solution and fails to display the timesteps.
The tests went through just fine with make -j4 and ./run_tests -j4, but running them individually is failing. Any suggestions? Thanks!

**Comment by ghost** [comment, marked as answer]
I figured it out

**Reply by cticenhour** [reply]
Happy to hear that you've figured it out! Could you add a quick comment with your solution just in case it could help others?

**Reply by ghost** [reply]
Yes; I simply made sure I used the run command listed in the README file listed in the "Example of running a Zapdos input file" section and viewed the results on paraview.

**Reply by cticenhour** [reply]
Thank you - that's my preferred method of visualization, but we would like to have Peacock available for users as well. I'll have to make a reminder to check out the specific issue with Peacock at a later time, as I thought they properly interpreted those parsed input file variables. If you'd like to make an issue regarding that to help us out (including the input file you tried, operating system version, etc), feel free to do so at https://github.com/shannon-lab/zapdos/issues
